src/js/tech/ChromecastTech.js

```diff
@@ -653,7 +653,7 @@ ChromecastTech = {
       var eventTypes = cast.framework.RemotePlayerEventType;
 
       this._addEventListener(this._remotePlayerController, eventTypes.MEDIA_INFO_CHANGED, () => {
-         if (typeof this._getMediaSession().activeTrackIds[0] === 'number') {
+         if (typeof this._getMediaSession()?.activeTrackIds[0] === 'number') {
```

Comment:
I'm not sure if this project's build process compiles down `?.` for older browsers

Reply:
https://github.com/standardbroadcast/videojs-chromecast/pull/10/commits/94d725f18f27cc7176a2a4849d18e4224b7a31b9